src/plots/plots.js

```diff
@@ -1524,6 +1527,21 @@ plots.supplyLayoutGlobalDefaults = function(layoutIn, layoutOut, formatObj) {
     coerce('modebar.activecolor', Color.addOpacity(modebarDefaultColor, 0.7));
     coerce('modebar.uirevision', uirevision);
 
+    coerce('newshape.drawdirection');
```

Comment:
These too would be nice to move to `components/shapes` - and referenced with `Registry.getComponentMethod`.

Reply:
Good call. Addressed in https://github.com/plotly/plotly.js/pull/4775/commits/63ef7aaf311858101c7ed3075a40472eb6341cd1.